### ISSUE TITLE
Position is shouted as rapidly as possible

### ISSUE THREAD

**Issue by Hoolean** [Stansbridge/untangled-2017#35]
It may be preferable to only shout the player's position when they move. Thoughts?

**Comment by fusspawn** [comment]
Yes. Best practice is usually to only send movements when a certain distance thresh hold is hit. (would just be one tile in your case). 

Its worth broadcasting once a second or so as a just in case situation.
